rest_framework_json_api/serializers.py

```diff
@@ -29,10 +29,8 @@ class ResourceIdentifierObjectSerializer(BaseSerializer):
 
     def __init__(self, *args, **kwargs):
         self.model_class = kwargs.pop('model_class', self.model_class)
-        if 'instance' not in kwargs and not self.model_class:
-            raise RuntimeError(
-                'ResourceIdentifierObjectsSerializer must be initialized with a model class.'
-            )
+        # this has no fields but assumptions are made elsewhere that self.fields exists.
+        self.fields = {}
```

Comment:
@sliverc I added test for OPTIONS but note that I  had to dummy up `self.fields` for it to work as assumptions are made elsewhere that views are either Model-based or otherwise have `fields` (also known as DJA "attributes"), which is not the case for a ResourceIdentifierSerializer which only has `type` and `id`.

Reply:
This should be OK.